### PR TITLE
Posit Workbench Acceptable Use Policy

### PR DESCRIPTION
**Pull request by terrymclaughlin**
This is still draft, but I wanted to make you both aware of this, in case you would like to contribute to its development?

### CODE REVIEW

File: posit_workbench_acceptable_use_policy.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Public Health Scotland have a fixed number of named-user licenses for Posit Workbench, and therefore users accessing Posit Workbench will be tracked for the sole purpose of ensuring compliance with the license agreement.
+
+## Appropriate Usage
+
+### Cloud Services
+
+Posit Workbench is hosted in Microsoft Azure, a cloud computing platform.  Microsoft Azure is a pay-as-you-go platform, and therefore most activities incur cost to Public Health Scotland, particularly running a Workbench session (even when it is idle) and any data being saved out of the Microsoft Azure environment e.g. saving to a file share on the Stats server.
+
+Users must take steps to minimise these costs by ensuring that they only request the computing resources they need to carry out their work, and ensuring that their analyses are as efficient as possible at all times.
```
Comment (ciarag01) [inline posit_workbench_acceptable_use_policy.md:39]: Should we link to the guidance on choosing a session size here? Or is there other guidance about writing efficient code that would be more helpful?

https://github.com/Public-Health-Scotland/R-Resources/blob/terrymclaughlin_posit_workbench_and_kubernetes/posit_workbench_and_kubernetes.md

File: posit_workbench_acceptable_use_policy.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+User activity on Posit Workbench will be routinely collated in dashboards and reports solely for the purposes of
+
+* monitoring the stability and performance of the Posit Workbench environment hosted in Microsoft Azure, and
+* calculating costs accrued by usage of Posit Workbench.
+
+Access to these audit logs, dashboards and reports will be restricted to authorised staff.  Authorisation shall be granted by ...
+
+## Training and Awareness
+
+Users can access appropriate training on the usage of Posit Workbench, R programming, GitHub and other Data Science topics related to use of Posit Workbench through the Data Science Knowledge Base.
```
Comment (ciarag01) [inline posit_workbench_acceptable_use_policy.md:60]: I think we should link to the Knowledge Base here for ease of access:

https://public-health-scotland.github.io/knowledge-base/

**Comment by ciarag01** [comment]
I've added a few comments @terrymclaughlin but other than those I think it looks good so far!

File: posit_workbench_acceptable_use_policy.md (hunk 1 above)
Comment (terrymclaughlin) [reply posit_workbench_acceptable_use_policy.md:43]: Done.

File: posit_workbench_acceptable_use_policy.md (hunk 2 above)
Comment (terrymclaughlin) [reply posit_workbench_acceptable_use_policy.md]: Done.

**Review by rmccreath — APPROVED** [review]
Hey @terrymclaughlin - sorry, I have a couple of broad things to mention, nothing that will necessarily stop this going live. Should we have a reference to the wider IT Acceptable Usage Policy and note that this is a "sub-policy"? By doing that we may be able to avoid the next thing which is to include more general exclusions (such as authorised business purposes, user responsibility for data and analytical integrity, or not causing harm through use of the system) and of who to report violations or ask questions?

**Comment by terrymclaughlin** [comment]
> Hey @terrymclaughlin - sorry, I have a couple of broad things to mention, nothing that will necessarily stop this going live. Should we have a reference to the wider IT Acceptable Usage Policy and note that this is a "sub-policy"? By doing that we may be able to avoid the next thing which is to include more general exclusions (such as authorised business purposes, user responsibility for data and analytical integrity, or not causing harm through use of the system) and of who to report violations or ask questions?

@rmccreath I agree with this.  I don't have a link to the NSS IT Acceptable Use Policy, but will reference it as the parent policy to this one.

**Pull request merged**